sources/packages/backend/apps/db-migrations/src/sql/StudentAssessments/Rollback-update-student-assessment-status.sql

```diff
@@ -0,0 +1,11 @@
+/*
+ Rollback assessments status updates.
+ The workflow is already changing the status to 'In progress' and 'Completed',
+ anything different can be considered as the default column value 'Submitted'.
+ */
+UPDATE
+  sims.student_assessments
+SET
+  student_assessment_status = 'Submitted'
+WHERE
+  student_assessment_status NOT IN ('In progress', 'Completed')
```

Comment:
I just wanted to double-check, During rollback, should we also consider `Cancelled` status?

Reply:
There is nothing right now in the solution that would change the assessment status to `Cancelled`. It will be introduced in upcoming tickets.

Reply:
Sorry, my bad, I completely misunderstood the point, and thanks for the conversation. I will add the `Cancelled` status to the rollback script.